packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -592,4 +592,71 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
   }
 
   // KHR_materials_iridescence
+  get iridescenceFactor(): number {
+    this[$ensureMaterialIsLoaded]();
+    return this[$backingThreeMaterial].iridescence;
+  }
+
+  get iridescenceTexture(): TextureInfo {
+    this[$ensureMaterialIsLoaded]();
+    return this[$pbrTextures].get(TextureUsage.Iridescence)!;
+  }
+
+  get iridescenceIor(): number {
+    this[$ensureMaterialIsLoaded]();
+    return this[$backingThreeMaterial].iridescenceIOR;
+  }
+
+  get iridescenceThicknessMinimum(): number {
+    this[$ensureMaterialIsLoaded]();
+    return this[$backingThreeMaterial].iridescenceThicknessRange[0];
+  }
+
+  get iridescenceThicknessMaximum(): number {
+    this[$ensureMaterialIsLoaded]();
+    return this[$backingThreeMaterial].iridescenceThicknessRange[1];
+  }
+
+  get iridescenceThicknessTexture(): TextureInfo {
```

Comment:
I just noticed this function signature doesn't match the API: one returns `TextureInfo`, while the other returns `TextureInfo|null`. I noticed because of the `!` below. This is probably true of the rest as well - let's make sure we know which is correct and then make them match.

Reply:
Have these been added to the pbr map with `createTextureInfo`? Maybe we need something more type-safe than a map...

Reply:
yes, these are all added at $initialize